Kernel/FileSystem/DevLoopFS/FileSystem.cpp

```diff
@@ -13,7 +13,7 @@
 
 namespace Kernel {
 
-ErrorOr<NonnullRefPtr<FileSystem>> DevLoopFS::try_create(ReadonlyBytes)
+ErrorOr<NonnullRefPtr<FileSystem>> DevLoopFS::try_create(ProcessID, ReadonlyBytes)
```

Comment:
Are we guaranteed to not needing to pass `ProcessID` to block based filesystems? is FUSE always about "non-block" based FSes in its nature due to being connected to a user program?